.github/workflows/workflow-build-macos.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
         git config --global init.defaultBranch master
     - name: Checkout repository code
       uses: actions/checkout@v3
+      with:
```

Comment:
These fetch-depth changes do not seem to belong in this commit. Why are they included?

Reply:
Because git describe will generate `-none` version info without them.